### PR TITLE
Add Tizen 6.5 bindings

### LINKED ISSUE #8: Add Tizen 6.5 binding APIs
- Add Tizen 6.5 binding APIs
- https://docs.tizen.org/application/native/api/iot-headed/6.5/index.html

### PR DESCRIPTION
**Pull request by WonyoungChoi**
close #8

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,4 +76,7 @@ build/
 !**/ios/**/default.perspectivev3
 
 # Tizen Rootstraps
-rootstraps/
\ No newline at end of file
+rootstraps/
+
+# Docs
+docs/**/*.png
```
Comment (swift-kim) [inline .gitignore:82]: There are only `.drawio` files in the directory. How do they work?

Comment (WonyoungChoi) [reply .gitignore:82]: The .drawio files are just diagrams that can be viewed with https://draw.io.

Comment (WonyoungChoi) [reply .gitignore:82]: https://app.diagrams.net/#HWonyoungChoi%2Ftizen_interop%2Fadd_6.5_bindings%2Fdocs%2Fdiagrams%2Fmodule_view.drawio

Comment (swift-kim) [reply .gitignore:82]: Will the diagrams be used anywhere in doc or are they just for who are interested?

Comment (WonyoungChoi) [reply .gitignore:82]: Only people interested in "internal module view".

**Review by swift-kim — APPROVED** [review]
Tested and worked. Thanks!

**Comment by WonyoungChoi** [comment]
Merge early because of other project dependence.

**Pull request merged**